packages/genesys-spark-components/src/components/stable/gux-rating/example.html

```diff
@@ -8,11 +8,19 @@ <h2>Defaults</h2>
 >
 </gux-rating>
 
+<h2>Integers only</h2>
```

Comment:
If the default is integers only, wouldn't it be better to give an example of half step increment instead?